src/pages/Admin/Venue/TemplateCard/TemplateCard.scss

```diff
@@ -0,0 +1,73 @@
+@import "scss/constants";
+
+// @debt Replace magic numbers with variables/constants/functions having meaningful names
```

Comment:
[OOS] Replacing all magic numbers with appropriate named variables would require more work and research going outside the scope of this PR, so only minor style changes made (extracting separate SCSS file, classes named per BEM)